sdcm/utils/sstable/load_utils.py

```diff
@@ -228,3 +244,24 @@ def get_load_test_data_inventory(cls, column_number: int, big_sstable: bool,
             return MULTI_COLUMNS_DATA
 
         return []
+
+    @classmethod
+    def create_keyspace(cls, node, keyspace_name: str = "keyspace1", strategy: str = 'SimpleStrategy',  # pylint: disable=too-many-arguments
+                        replication_factor: int = 1, drop_ks_if_exists: bool = False):
+        if drop_ks_if_exists:
+            node.run_cqlsh(f"DROP KEYSPACE {keyspace_name}")
+
+        node.run_cqlsh("""
+                CREATE KEYSPACE %s WITH replication = {'class': '%s',
+                'replication_factor': %s}
+                """ % (keyspace_name, strategy, replication_factor))
+
+    @classmethod
+    def create_table_for_load(cls, node, schema_file_and_path: str):
+        schema = node.remoter.run(f"cat {schema_file_and_path}").stdout
```

Comment:
nitpick, 
it would be nice to have `IF NOT EXISTS` in the table creation as well.
but it's a bit trickier, since the statement needs to be altered

Reply:
I did it in intention. I want to drop keyspace if it exists.
It should be appropriate schema that selected snapsot needs